package.json

```diff
@@ -36,7 +37,9 @@
     "moment": "^2.22.2",
     "ora": "^3.0.0",
     "prompt-confirm": "^2.0.4",
+    "typedarray-to-buffer": "^3.1.5",
```

Comment:
why is this needed?

Reply:
It crashed and required to install that for some reason